packages/paste-core/components/form-pill-group/src/FormPill.styles.ts

```diff
@@ -0,0 +1,87 @@
+import type {BoxStyleProps} from '@twilio-paste/box';
+
+/*
+ * Default variant styles
+ */
+
+export const defaultBaseStyles: BoxStyleProps = {
+  color: 'colorText',
+  backgroundColor: 'colorBackgroundPrimaryWeakest',
+
+  _focus: {
+    boxShadow: 'shadowFocus',
+    color: 'colorTextLinkStronger',
+  },
+  _selected: {
+    backgroundColor: 'colorBackgroundPrimaryStronger',
+    color: 'colorTextWeakest',
+  },
+  _selected_focus: {
+    boxShadow: 'shadowFocus',
+    color: 'colorTextInverse',
+  },
+  _disabled: {
+    backgroundColor: 'colorBackgroundStrong',
+    cursor: 'not-allowed',
+    color: 'colorText',
+  },
+};
+
+export const defaultHoverStyles: BoxStyleProps = {
+  cursor: 'pointer',
+  _hover: {
+    color: 'colorTextLinkStronger',
+    borderColor: 'colorBorderPrimaryStronger',
+  },
+  _selected_hover: {
+    backgroundColor: 'colorBackgroundPrimary',
+    borderColor: 'transparent',
+    color: 'colorTextInverse',
+  },
+  _focus_hover: {
+    borderColor: 'transparent',
+  },
+};
+
+/*
+ * Error variant styles
+ */
+export const errorBaseStyles: BoxStyleProps = {
+  backgroundColor: 'colorBackgroundErrorWeakest',
+  color: 'colorTextError',
+
+  _focus: {
+    boxShadow: 'shadowFocus',
+    color: 'colorTextErrorStrong',
+  },
+  _selected: {
+    backgroundColor: 'colorBackgroundError',
+    color: 'colorTextInverse',
+  },
+  _selected_focus: {
+    boxShadow: 'shadowFocus',
+    color: 'colorTextInverse',
+  },
+  _disabled: {
+    backgroundColor: 'colorBackgroundStrong',
+    cursor: 'not-allowed',
+    color: 'colorText',
+  },
+};
+
+export const errorHoverStyles: BoxStyleProps = {
+  cursor: 'pointer',
+  _hover: {
+    borderColor: 'colorBorderErrorStronger',
+    color: 'colorTextErrorStronger',
+  },
+  _selected_hover: {
+    backgroundColor: 'colorBackgroundErrorStrongest',
+    borderColor: 'transparent',
+    color: 'colorTextInverse',
+  },
+};
+
+// todo
+// text color on hover on errro
+// border color should be transparent on focus_hover
```

Comment:
TODO: 
- Review and fix the text color on hover for the error state. It should use the new token listed in the ticket on the Close icon and the FormPillButton content too.
- When focusing and hovering, the border shouldn't be visible. I had to create a new pseudoProp on Box for this called `_focus_hover`, but I'm not sure it's working as I didn't have time to test it yet. I added it into line 41 in this file just in case it works on the first try.